source/SIL.AppBuilder.Portal.Frontend/package.json

```diff
@@ -71,8 +71,8 @@
     "react-notify-toast": "^0.5.0",
     "react-orbitjs": "github:DeveloperTown/react-orbitjs#9913bdeac3495a9f75413c30f168ed617cf29124",
     "react-redux": "^6.0.0",
-    "react-router": "^4.3.1",
-    "react-router-dom": "^4.3.1",
+    "react-router": "^4.4.0-beta.6",
```

Comment:
in order to get access to the router context via private api to make a `useRouter` hook.